tests/conftest.py

```
@@ -0,0 +1,77 @@
+        'mz_array': [100.0, 200.0, 300.0],
+        'intensity_array': [1000.0, 2000.0, 3000.0],
+        'peptidoform': 'PEPTIDE',
+        'posterior_error_probability': 0.01,
+        'Nreps': 5
+    }
+
+
+@pytest.fixture
+def sample_cluster_df():
+    """Create a sample cluster DataFrame."""
+    import numpy as np
```

Ruff (0.14.10)

Redefinition of unused `np` from line 7

Remove definition: `np`

(F811)